cardshifter-server/src/main/java/com/cardshifter/server/model/MainServer.java

```diff
@@ -71,6 +71,11 @@ public MainServer(ServerConfiguration serverConfiguration) {
 	public Server start() {
         mods.loadExternal(mods.getDefaultModLocation());
         mods.loadExternal(Paths.get(config.getModsDirectory()));
+
+        if(mods.getAvailableMods().isEmpty()) {
+        	logger.warn("No mods were loaded from the different folders. Please check your installation.");
```

Comment:
As far as I can tell, that's only logged if the directory does not exist. ´directory` could still point to a valid directory with no mods inside it.